doc/supportedPlatformVerions.md

```diff
@@ -0,0 +1,949 @@
+# Supported platforms and versions
```

Comment:
I think we can remove this misspelled file in favor of the correctly named one thats modified in this PR

Reply:
Updated